docs/README.md

```diff
@@ -1,4 +1,4 @@
 README
 ======
 
-This file is a placeholder. This directory will contain public documentation.
+This file is a placeholder.  This directory will contain public documentation.
```

Comment:
I don't think we had a standard. @dpoeschl might have been referring to the widely held opinion that double spaces are a holdover from the typewriter days.